### PR TITLE
15 front end form for user log in

### PR DESCRIPTION
**Pull request by lyremlou**
with some back end updates for /register and /login

### CODE REVIEW

**Review by jlacy93 — APPROVED** [review]

File: feral-cat-tracker-UI/src/app/app.module.ts [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
 import { HomeComponent } from './home/home.component';
+import { LoginComponent } from './login/login.component';
+import { HttpClientModule } from  '@angular/common/http';
 @NgModule({
   declarations: [
     AppComponent,
     HeaderbarComponent,
     NavbarComponent,
     FooterbarComponent,
+    LoginComponent
```
Comment (Meliora347) [inline feral-cat-tracker-UI/src/app/app.module.ts:19]: Needs a "," following LoginComponent

File: feral-cat-tracker-UI/src/app/app.module.ts [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
     ContactComponent,
     HomeComponent
   ],
   imports: [
     BrowserModule,
     AppRoutingModule,
+    ReactiveFormsModule,
+    HttpClientModule
```
Comment (Meliora347) [inline feral-cat-tracker-UI/src/app/app.module.ts:27]: Needs a "," following HttpClientModule

**Review by Meliora347 — CHANGES_REQUESTED** [review]
Needs commas added in app.module.ts following LoginComponent and HttpClientModule. It currently won't run, but WILL run when I make these changes.

File: feral-cat-tracker-UI/src/app/app.module.ts (hunk 1 above)
Comment (lyremlou) [reply feral-cat-tracker-UI/src/app/app.module.ts]: Got it!

**Review by Meliora347 — APPROVED** [review]

**Pull request merged**